### DISCUSSION TITLE
Pre built images for MemcardPro2

### DISCUSSION THREAD

**Discussion by israpps** [israpps/FreeMcBoot-Installer#32] [General]
Pre built images of all the FreeMcBoot versions hosted here to be used with the MemcardPro2 of 8bitmods

<hr /><em>This discussion was created from the release <a href='https://github.com/israpps/FreeMcBoot-Installer/releases/tag/mcpro2-img'>Pre built images for MemcardPro2 </a>.</em>

**Comment by Priceless37** [comment]
any chance of you remaking these now including the Ulaunchelf and OPL versions that support MMCE? would be wonderful if you did as most of us are intending on using mmce with our mempros and this would make it even easier. thank you regardless.
